commands/import/index.js

```diff
@@ -237,7 +237,9 @@ class ImportCommand extends Command {
           tracker.completeWork(1);
         })
         .catch(err => {
-          if (err.stdout.indexOf("Patch is empty.") === 0) {
+          const patchIsEmptyMessages = ["Patch is empty.", "Патч пустой."];
```

Comment:
Is there any other way of detecting this status _independently_ of an arbitrary localized string?

Reply:
I found a better way that works without hard-coded list of strings.

Reply:
@evocateur I reviewed your request change and updated code long time ago. Is there anything else I should do?